seqr/views/apis/family_api_2_3_tests.py

```diff
@@ -188,7 +190,7 @@ def test_receive_families_table_handler(self):
         self.assertEqual(response.status_code, 200)
         response_json = response.json()
 
-        self.assertListEqual(response_json.keys(), ['info', 'errors', 'warnings', 'uploadedFileId'])
+        self.assertSetEqual(set(list(response_json.keys())), {'info', 'errors', 'warnings', 'uploadedFileId'})
```

Comment:
you sholdn't need to do `set(list(...))`, just `set(...)` should work

Reply:
You are right. Removed it.